action.go

```diff
@@ -670,3 +673,61 @@ func makeCombinedAction(actions ...Action) ActionFunc {
 		})
 	})
 }
+
+func makeCommandAction(cc *CommandConfig) ActionFunc {
+	return func(i *Input, _ termbox.Event) {
+		if i.SelectionLen() == 0 {
+			i.SelectionAdd(i.currentLine)
+		}
+
+		i.selection.Ascend(func(it btree.Item) bool {
+			line := it.(Line)
+
+			var f *os.File
+			var err error
+
+			args := append([]string{}, cc.Args...)
+			for i, v := range args {
+				switch v {
+				case "$FILE":
+					if f == nil {
+						f, err := ioutil.TempFile("", "peco")
+						if err != nil {
+							return false
+						}
+						f.WriteString(line.Buffer())
+						f.Close()
+					}
+					args[i] = f.Name()
```

Comment:
If `if` block is executed, then `f.Name()` is called after `f.Close()`. This causes null pointer reference.